changelog.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+ - [[#1438] Add SFTP and related security](https://github.com/ballerina-platform/ballerina-standard-library/issues/1438)
+
```

Comment:
* As per the changelog discussion, fixes/new-features/improvements done should be in unreleased section until the release is done. @shafreenAnfar wdyt ?

Reply:
AFAIR, we can have `Added`, `Changed` etc. under the `[Unreleased]` heading. @shafreenAnfar please correct me if I am wrong.